workers/unity/Packages/io.improbable.gdk.buildsystem/Configuration/BuildConfigEditor.cs

```diff
@@ -710,13 +715,27 @@ private void DrawBuildTargets(BuildEnvironmentConfig env, int hash)
                     RecordUndo("Worker build options");
 
                     env.BuildTargets[selectedBuildTarget.Index] =
-                        new BuildTargetConfig(buildTarget.Target, options, enabled, required);
+                        new BuildTargetConfig(buildTarget.Target, options, enabled, required, scriptingImplementation);
 
                     selectedBuildTarget.Choices = null;
                 }
             }
         }
 
+        private ScriptingImplementation ConfigureScriptingImplementation(ScriptingImplementation backend, BuildTarget buildTarget)
+        {
+            var choice = backend;
```

Comment:
I think you can remove this local declaration (i.e. - always use the `backend` binding